frontend/src/scenes/plugins/pluginActivityDescriptions.tsx

```diff
@@ -169,5 +169,48 @@ export function pluginActivityDescriber(logItem: ActivityLogItem): HumanizedChan
         }
     }
 
+    if (logItem.activity.startsWith('attachment_')) {
```

Comment:
Request: For future FE changes, let's also include screenshots!

Reply:
will do!